exporter/coralogixexporter/config.go

```diff
@@ -73,11 +79,11 @@ func isEmpty(endpoint string) bool {
 }
 func (c *Config) Validate() error {
 	// validate that at least one endpoint is set up correctly
-	if isEmpty(c.Endpoint) &&
+	if isEmpty(c.Domain) &&
 		isEmpty(c.Traces.Endpoint) &&
 		isEmpty(c.Metrics.Endpoint) &&
 		isEmpty(c.Logs.Endpoint) {
-		return fmt.Errorf("`traces.endpoint` or `metrics.endpoint` or `logs.endpoint` not specified, please fix the configuration file")
+		return fmt.Errorf("`domain` or `traces.endpoint` or `metrics.endpoint` or `logs.endpoint` not specified, please fix the configuration file")
```

Comment:
```suggestion
		return fmt.Errorf("`domain` or `traces.endpoint` or `metrics.endpoint` or `logs.endpoint` not specified, please fix the configuration")
```

Technically the collector's configuration can come from non-file inputs.